app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -83,10 +90,27 @@ class EditProfileFragment: DialogFragment() {
         super.onResume()
 
         val editProfileDialog = dialog as AlertDialog
+
+        mentorSwitch = editProfileBinding.switchAvailableToMentor.findViewById(R.id.switchAvailableToMentor)
```

Comment:
keep the two assignment statements together.
and 
keep the listener code together.

Reply:
```
mentorSwitch = editProfileBinding.switchAvailableToMentor.findViewById(R.id.switchAvailableToMentor)
menteeSwitch = editProfileBinding.switchNeedsMentoring.findViewById(R.id.switchNeedsMentoring)

mentorSwitch.setOnCheckedChangeListener { buttonView, isChecked ->
            editProfileBinding.user?.isAvailableToMentor = isChecked
        }
menteeSwitch.setOnCheckedChangeListener { buttonView, isChecked ->
            editProfileBinding.user?.needsMentoring = isChecked
        }
```

Change it to this